### PR TITLE
Catch the bad status line exception and log it

### PR DESCRIPTION
**Pull request by laxdog**
Polo sometimes return a bad error code we can't parse. This catches that exception. 

## Description
Fixes https://github.com/Mikadily/poloniexlendingbot/issues/205

## TESTING STAGE
Tested

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] **I have read CONTRIBUTING.md**
- [x] **I fully understand [Github Flow.](https://guides.github.com/introduction/flow/)**
- [x] **My code adheres to the [code style of this project.](https://poloniexlendingbot.readthedocs.io/en/latest/contributing.html)**
- [N/A] **I have updated the documentation in /docs if I have changed the config, arguments, logic in how the bot works, or anything that understandably needs a documentation change.**
- [N/A] **I have updated the config file accordingly if my change requires a new configuration setting or changes an existing one.**
- [x] **I have tested the bot with no issues for 24 continuous hours. If issues were experienced, they have been patched and tested again.**

### CODE REVIEW

**Comment by rnevet** [comment]
1. It's not so clear why this needs special handling. (to prevent sleep?)
2. I'm not sure the handling is complete without persisting the status. (i.e. log.persistStatus())

**Comment by laxdog** [comment]
1) No reason. I actually thought they were separated because the first ones were catching the class and the others were using the string. 
2) I didn't realise that was required either. 

Changed and pushed.

**Comment by rnevet** [comment]
Looks good to me, a bit of testing and it can be merged.

File: lendingbot.py [hunk 1]
```diff
@@ -94,6 +95,8 @@
                 exit(1)
             elif 'timed out' in ex.message:
                 print "Timed out, will retry in " + str(Lending.get_sleep_time()) + "sec"
+            elif isinstance(ex, BadStatusLine):
```
Comment (chfast) [inline lendingbot.py]: Should it go as a `except BadStatusLine:`?

Comment (rnevet) [inline lendingbot.py]: It could but then we would need to duplicate the entire error logic that is in `except Exception:` already.

**Pull request merged**